graphql-parser/src/lib.rs

```diff
@@ -127,7 +127,7 @@
 //! let mut fields = Fields { output: vec![] };
 //! ast.accept(&mut fields);
 //! assert_eq!(fields.output, vec!["fieldA", "fieldB", "fieldC"]);
-//!# Ok::<(), schema::ParseError>(())
+//!# Ok::<(), graphql_parser::ParseError>(())
```

Comment:
Hm so I think these lines are not required? I found out that the `#` thing only hides that line from the docstring, but still runs that code. In this case (and in line 180) we don't have a `fn` that returns `Result<(), ParseError>` like in line 49, so this code is run as if it's part of a `main` (I'm pretty sure), which means it has no return type.

Reply:
https://doc.rust-lang.org/rustdoc/documentation-tests.html#using--in-doc-tests